tests/benchdnn/graph/graph.cpp

```diff
@@ -407,11 +407,6 @@ void skip_unimplemented_ops(const dnnl::graph::partition &partition,
         const deserialized_graph_t &dg, res_t *res) {
     // A list of ops that don't have DNNL backend support so far.
     static const std::vector<std::string> unimplemented_ops {"Pow"};
-    // A list of ops that don't have DNNL backend support so far on GPU.
-    static const std::vector<std::string> unimplemented_ops_gpu {
-            "GreaterEqual"};
```

Comment:
I'd just remove the op from `unimplemented_ops_gpu` and kept rest infrastructure intact for future unsupported ops.